src/services/pool/exchange/serializers/ExitParams.ts

```diff
@@ -78,10 +100,22 @@ export default class ExitParams {
 
   private parseTokensOut(tokensOut: string[]): string[] {
     const nativeAsset = this.config.network.nativeAsset;
-
-    return tokensOut.map(address =>
+    const poolTokenItselfIndex = preMintedBptIndex(this.pool.value);
```

Comment:
May as well rename this const too:
```suggestion
    const preMintedBptIndex = preMintedBptIndex(this.pool.value);
```